demos/spectre_v1_btb_sa.cc

```diff
@@ -123,9 +124,7 @@ static char LeakByte(size_t offset) {
       // We make sure to flush whole accessor object in case it is
       // hypothetically on multiple cache-lines.
       const char *accessor_bytes = reinterpret_cast<const char*>(accessor);
-      for (size_t j = 0; j < object_size_in_bytes; j += kCacheLineSize) {
-        CLFlush(accessor_bytes + j);
-      }
+      FlushFromCache(accessor_bytes, accessor_bytes + object_size_in_bytes);
```

Comment:
These might be stray changes from a different branch

Reply:
This concrete change is intentional. I'm using the new FlushFromCache function in utils (introduced for the recursive ret2spec) to flush whole accessor object from the cache (even though the virtual pointer is probably always on the beginning of the object).